README.md

```diff
@@ -1,2 +1,26 @@
-# gpii-devpmt
-Development Preferences Test Tool and other Management Tools
+# Development Preferences Test Tool and other Management Tools
+
+This project is the foundation for the DPTT and other comprehensive preferences
+management tools for the GPII.
+
+# Installation
+
+```bash
+git clone https://github.com/sgithens/gpii-devpmt.git
+cd gpii-devpmt
+yarn install # npm probably works too
```

Comment:
Until issues such as https://github.com/yarnpkg/yarn/issues/2496 are fixed, our project recommendation is to continue with npm

Reply:
Documentation/README has been updated to use npm